### DISCUSSION TITLE
Full syntax list

### DISCUSSION THREAD

**Discussion by xNONITIMEx** [leonhma/obsidian-functionplot#145] [Q&A]
Hey, I like your plugin so much, but maybe you have a more complete syntax list, bcs I wonder if I can do something more than plotting simple graphs. Thank you!

**Comment by FennexFox** [comment]
It would really be helpful

**Comment by leonhma** [comment]
The evaluation and plotting is actually done by an underlying package. Take a look at the wiki: https://github.com/leonhma/obsidian-functionplot/wiki/Supported-function-syntax
It might even be possible that new functions have been added by the underlying library in the meantime and the wiki is lagging behind. I recommend take a look at the functionplot docs if something is missing or raise an issue over there.
